zerver/models.py

```diff
@@ -1588,6 +1588,13 @@ class UserBaseSettings(models.Model):
     notification_sound = models.CharField(max_length=20, default="zulip")
     wildcard_mentions_notify = models.BooleanField(default=True)
 
+    # Followed Topics notifications.
+    enable_followed_topic_desktop_notifications = models.BooleanField(default=False)
+    enable_followed_topic_email_notifications = models.BooleanField(default=False)
+    enable_followed_topic_push_notifications = models.BooleanField(default=False)
+    enable_followed_topic_audible_notifications = models.BooleanField(default=False)
+    enable_followed_topic_wildcard_mentions_notify = models.BooleanField(default=True)
```

Comment:
Why not end this with `notifications` (instead of `notify`) like the other fields?

Reply:
I did this to keep it consistent with the existing:

```python
    # Stream notifications.
    enable_stream_desktop_notifications = models.BooleanField(default=False)
    enable_stream_email_notifications = models.BooleanField(default=False)
    enable_stream_push_notifications = models.BooleanField(default=False)
    enable_stream_audible_notifications = models.BooleanField(default=False)
    notification_sound = models.CharField(max_length=20, default="zulip")
    wildcard_mentions_notify = models.BooleanField(default=True)
```

Reply:
Yeah, consistent is probably correct. It is a significantly different behavior, since it's whether wildcard mentions are eligible for notifications at all.